src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/ResourceActions/ResourceActions.ts

```diff
@@ -16,6 +16,7 @@ export var resourceActionsDirective = (
         scope: {
             resourcePath: "@",
             parentPath: "=?",
+            deleteRedirectUrl: "@?",
```

Comment:
this parameter will be used for a future `adhDeleteAction` as well, which is why it's called `deleteRedirectUrl`.